easybuild/easyconfigs/i/ImageJ/ImageJ-1.52q-Java-1.8.eb

```diff
@@ -0,0 +1,42 @@
+easyblock = 'PackedBinary'
+
+name = 'ImageJ'
+version = '1.52q'
+versionsuffix = '-Java-%(javaver)s'
+
+homepage = 'https://imagej.nih.gov/ij'
+description = "Image Processing and Analysis in Java"
+
+toolchain = SYSTEM
+
+source_urls = [
+    'https://imagej.nih.gov/ij/download/src/',
+    # 'http://www.mecourse.com/landinig/software/',  # old/broken
+    'https://beardatashare.bham.ac.uk/getlink/fiLUS55SmtWwJKijDRbwqv9p/'
+]
+sources = [
+    'ij%(version_major)s%(version_minor)s-src.zip',
+    'morphology.zip',
```

Comment:
I think it makes sense to rename this on download, to avoid clashes with potential other `morphology.zip` plugins (updated versions for example).

How about this:

```python
sources = [
    'ij%(version_major)s%(version_minor)s-src.zip',
    # datestamp is determined by most recent file in zipball
    {'download_filename': 'morphology.zip', 'filename': 'morphology-20190912.zip'},
]

Reply:
@JackPerdue Any updates?

Reply:
I think @boegel fixed this is https://github.com/easybuilders/easybuild-easyconfigs/pull/9245    I guess I can close this one

Reply:
Well, I only fixed the easyconfig for ImageJ 1.51k, I did not add an easyconfig for version 1.52q...